lib/experimental/new_embed.dart

```diff
@@ -607,6 +612,43 @@ major browsers, such as Firefox, Edge (dev channel), or Chrome.
     setContextSources(lastInjectedSourceCode);
   }
 
+  void _handleCopyCode() {
+    var textElement = document.createElement('textarea') as TextAreaElement;
+    textElement.value = _getActiveSourceCode();
+    document.body.append(textElement);
+    textElement.select();
+    document.execCommand('copy');
+    textElement.remove();
+  }
+
+  String _getActiveSourceCode() {
+    String activeSource;
+    String activeTab = "editor";
+
+    try {
+      // TODO: Remove this try-catch block
+      // by fixing the selectedTab to return first tab, if selected attr not available.
+      activeTab = tabController.selectedTab.name;
+    } catch (_) {}
+
+    switch (activeTab) {
```

Comment:
I believe this is missing two other possibilities: the test method and the solution.

Have a look at this URL in your build of the site:

`/embed-flutter.html?id=2234a5ccada200eb1e018b12fa95d57d`

There is an additional solution tab, and if you use the menu in the top right, you can enable the test tab as well.

Reply:
Considered solution and test tab also.